### PR TITLE
Action bar tweaks

### PR DESCRIPTION
**Pull request by JPLachance**
! Fix actions vertical align
! Fix actions padding

### CODE REVIEW

File: scss/tables/table-actions.scss [hunk 1]
```diff
@@ -27,7 +29,15 @@
         opacity: 0.3;
       }
 
+      & + .action {
+        margin-left: 30px;
+      }
+
       &.dropdown {
+        padding: 3px 10px;
+        margin-right: -10px;
```
Comment (pochretien) [inline scss/tables/table-actions.scss]: i think we can create a variable for this 10px removed

**Comment by pochretien** [comment]
:ship: :it:

**Comment by wfortin** [comment]
:+1:

File: scss/tables/table-actions.scss (hunk 1 above)
Comment (JPLachance) [inline scss/tables/table-actions.scss]: Done!

**Pull request merged**